janus_client/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "janus_client"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
+rust-version = "1.58"
+
+[dependencies]
+http = "0.2.8"
+janus = { path = "../janus" }
+prio = "0.8.0"
+reqwest = { version = "0.11.4", default-features = false, features = ["rustls-tls"] }
+thiserror = "1.0"
+tokio = { version = "^1.19", features = ["full"] }
+url = "2.2.2"
+
+[dev-dependencies]
+assert_matches = "1"
+janus_test_util = { path = "../test_util" }
+mockito = "0.31.0"
+tracing = "0.1.34"
```

Comment:
I'm surprised that we only use `tracing` in test configs. Naturally we shouldn't pull in `tracing-log` and `tracing-subscriber`, but the production version of `janus_client` ought to be emitting trace events that clients can choose to subscribe to. IMO we should at least put a `#[tracing::instrument(..., err)]` on `Client::upload`, but I suppose that's beyond the scope of what this PR wants to accomplish.